NitroxClient/GameLogic/Item.cs

```diff
@@ -47,8 +47,18 @@ public void Dropped(GameObject gameObject, TechType techType, Vector3 dropPositi
 
             Log.Debug("Dropping item with guid: " + guid);
 
-            DroppedItem droppedItem = new DroppedItem(guid, waterpark, techType, dropPosition, bytes);
-            packetSender.Send(droppedItem);
+            if (techType == TechType.Beacon | techType == TechType.Pipe)
```

Comment:
Would we need to do a similar check for pickup item?

Reply:
yep